website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -4,391 +4,275 @@ linkTitle: Releasing
-through the sections directly following this one.
+These steps describe how to conduct a release of the operator-sdk repo using example versions.
+Replace these versions with the current and new version you are releasing, respectively.
 
-## Dependency and platform support
+## Prerequisites
 
-### Go version
+- [`git`][git]
+- [`gpg2`][gpg2] and a [GPG key][gpg-key-create]. If you have both `gpg` and `gpg2` available, make sure the latter is used:
+    ```sh
+    git config --global gpg.program gpg2
+    ```
+- Your GPG key is publicly available in a [public key server][gpg-upload], like https://keyserver.ubuntu.com/.
 
-Release binaries will be built with the Go compiler version specified in the Operator SDK's [prerequisites section][doc-readme-prereqs].
+##### MacOS users
 
-### Kubernetes versions
+Install GNU `sed`, `make`, and `gpg2` which may not be by default:
 
-As the Operator SDK interacts directly with the Kubernetes API, certain API features are assumed to exist in the target cluster. The currently supported Kubernetes version will always be listed in the SDK [prerequisites section][doc-readme-prereqs].
+```sh
+brew install gnu-sed make gnupg
+```
 
-### Operating systems and architectures
+Configure `gpg2` to sign with `git`:
 
-Release binaries will be built for the `x86_64` architecture for MacOS Darwin platform and for the following GNU Linux architectures: `x86_64`, `ppc64le`, `arm64`, and `s390x`.
+```bash
+echo "export GPG_TTY=tty" >> ~/.bashrc
+source ~/.bashrc
+```
 
-Base images for ansible-operator, helm-operator, and scorecard-test will be built for the following GNU Linux architectures: `x86_64`, `ppc64le`, and `arm64`.
+## Major and Minor releases
 
-Support for the Windows platform is not on the roadmap at this time.
+We will use the `v1.3.0` release version in this example.
 
-## Binaries and signatures
+### Before starting
 
-Binaries will be signed using a maintainers' verified GitHub PGP key. Both binary and signature will be uploaded to the release. Ensure you import maintainer keys to verify release binaries.
+A release branch must be created and [mapped][netlify-deploy] _before the release begins_
+to appease the Netlify website configuration demons. You can ping SDK [approvers][doc-owners] to ensure a
+[release branch](#release-branches) is created prior to the release and that this mapping is created.
+If you have the proper permissions, you can do this by running the following,
+assuming the upstream SDK is the `upstream` remote repo:
 
-## Release tags
+```sh
+git checkout master
+git pull
+git checkout -b v1.3.x
+git push -u upstream v1.3.x
+```
 
-Every release will have a corresponding git semantic version tag beginning with `v`, ex. `v1.2.3`.
+Also create and merge a commit that updates the top-level [Makefile] variable `IMAGE_VERSION`
+to the upcoming release tag `v1.3.0`. This variable ensures sample projects have been tagged
+correctly prior to the release commit.
```

Comment:
I guarantee I *will* miss this step. Is there a way to call it out in a command block?

```sh
sed -ie 's/IMAGE_VERSION = v[0-9]\.[0-9]\.[0-9]/IMAGE_VERSION = v1.3.0/' Makefile
```
Something even the following would work as a reminder for me:
```sh
vi Makefile
# EDIT IMAGE_VERSION
```

Basically what happens is first time through I'll read the doc, but subsequent times I look for the command blocks.